draft-wang-ppm-differential-privacy.md

```diff
@@ -456,9 +523,105 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
```

Comment:
nit: A better intro sentence would be: "This section described the binary randomized response mechanism."